server/room-battle.ts

```diff
@@ -726,6 +728,14 @@ export class RoomBattle extends RoomGames.RoomGame {
 		for (const player of this.players) player.wantsTie = false;
 
 		switch (lines[0]) {
+		case 'requestlog':
```

Comment:
`requestlog` and `requestedteam` can be the same part of this now. Maybe call it `requesteddata`?